### ISSUE TITLE
Channel ids for sorted units are missing after export_to_phy

### ISSUE THREAD

**Issue by MingChingChiang** [SpikeInterface/spiketoolkit#376]
Hi, 
I'm trying to export my klusta sorted units to phy.
Although the export is successfully, all the channel ids for each unit become 0. 

<img width="1680" alt="Screen Shot 2020-06-24 at 23 53 13" src="https://user-images.githubusercontent.com/63445768/85590354-e99d0780-b676-11ea-8934-d3c392593b9a.png">

I've double checked my sorting by opening the kwik file with phy directly, but each units have channel ids.

<img width="1680" alt="Screen Shot 2020-06-24 at 23 35 06" src="https://user-images.githubusercontent.com/63445768/85590155-c1150d80-b676-11ea-881c-256ec9131a5f.png">

I'm using the following code to export my sorting results.

`st.postprocessing.export_to_phy(recording_sub_ch, sorting_KL, output_folder=KL_Phy, compute_pc_features=True,
                                max_channels_per_template=None, recompute_info=True, compute_property_from_recording=True,
                                grouping_property='group', ms_before=2, ms_after=3, memmap=True, verbose=True)
`

Did I just miss the argument for the channel ids setting?
Or where can I fix the problem?

Thank you for your help!

**Comment by alejoe91** [comment]
Hi! 

This has been recently fixed by this PR: https://github.com/SpikeInterface/spiketoolkit/pull/375

Can you install spiketoolkit from the latest master and check that it's fixed?

**Comment by MingChingChiang** [comment]
> Hi!
> 
> This has been recently fixed by this PR: #375
> 
> Can you install spiketoolkit from the latest master and check that it's fixed?

Thanks! @alejoe91 

After reinstall the spiketoolkit now I can see channel ids.

<img width="1680" alt="Screen Shot 2020-06-25 at 13 16 26" src="https://user-images.githubusercontent.com/63445768/85658954-864abe00-b6e6-11ea-9938-61776460768e.png">

A minor question is that I removed many bad channels and only kept good channels for sorting.
Is there any way after export to phy I can see the original channel id instead of the ids starts from 0?

Thanks anyway!

**Comment by alejoe91** [comment]
@MingChingChiang glad that it works!

Unfortunately Phy forces the channels to be 0-N, but if you load back the results with SpikeInterface the correct channel Id will be used. Also the maximum channel from each unit can be obtained directly in SI with:

```
max_channels = st.postprocessing.get_unit_max_channels(recording, sorting)
```

Closing the issue :)
